examples/org-website/list_sample.go

```diff
@@ -1,46 +1,51 @@
+//go:build ignore
 // +build ignore
 
+/*
```

Comment:
Will the copy right comment be visible? If yes, I think we should remove them. It would a lot easier to read at a single glance.